core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -43,7 +43,22 @@ oppia.factory('AssetsBackendApiService', [
         responseType: 'blob',
         url: _getAudioDownloadUrl(explorationId, filename),
       }).success(function(data) {
-        var audioBlob = new Blob([data]);
+        try {
+          var audioBlob = new Blob([data]);
+        } catch (exception) {
+          window.BlobBuilder = window.BlobBuilder ||
+                         window.WebKitBlobBuilder ||
+                         window.MozBlobBuilder ||
+                         window.MSBlobBuilder;
+          if (exception.name == 'TypeError' && window.BlobBuilder) {
+            var blobBuilder = new BlobBuilder();
+            blobBuilder.append(data);
+            var audioBlob = blobBuilder.getBlob('audio/*');
+          } else {
+            throw new Error(
+              'Your browser does not support autogenerated audio.');
```

Comment:
Why just autogenerated audio? Isn't this for fetching audio files in general...?

Also, just do `throw exception;` to preserve the stack trace. Then, if we get any further reports from this part of the code, we'll know the fix hasn't worked. Otherwise you're just shadowing it.

Reply:
Done.